src/components/timeline/Task.vue

```diff
@@ -134,13 +134,19 @@ export default {
       this.updateTask(x, width);
     },
     onDrag(x) {
+      console.log("onDrag ");
       this.updateTask(x, this.width);
       this.dragging = false;
     },
     onDragStart() {
+      console.log("onDragStart ");
       this.dragging = true;
     },
     updateTask(x, width) {
+      console.log("updateTask ");
+      this.task.assignee = Number(sessionStorage.getItem("assigneeID")); //neue AssigneeId wird dem Task zugeordnet
+      this.editTask(this.task);
```

Comment:
A direct mutation of the component's prop seems a bit weird to me. A better approach would be to just commit the intended changes without mutating the original object and let the component re-render, as soon as the store updates. This could prevent bugs in the long run.

Additionally, the assignee will now be changed whenever the task was dragged - even if the assignee did actually not change. This does no real harm as it probably, won't cause any errors, but leads to unnecessary store updates and eventually additional renderings ==> unnecessary performance overhead.

```suggestion
      const assignee = Number(sessionStorage.getItem("assigneeID"));
      if (this.task.assignee !== assignee) {
        this.editTask({ ...this.task, assignee });
      }
```